src/UWP/ArcGISRuntime.UWP.Viewer/App.xaml.cs

```diff
@@ -39,22 +32,20 @@ public App()
             Suspending += OnSuspending;
             UnhandledException += App_UnhandledException;
             TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
-
-            ApplicationManager.Current.Initialize(SamplesLanguageUsed);
         }
 
         private void TaskScheduler_UnobservedTaskException(object sender, UnobservedTaskExceptionEventArgs e)
         {
             var errorMessage = e.Exception.ToString();
             Debug.WriteLine(errorMessage);
-            var message = new MessageDialog(errorMessage, "An error occurred").ShowAsync();
+            new MessageDialog(errorMessage, "An error occurred").ShowAsync();
```

Comment:
Warning CS4014 — async method call is not awaited. Consider making this handler async-void, to get rid of compiler error.